chromadb/api/segment.py

```diff
@@ -340,10 +343,17 @@ def _add(
         self._quota.static_check(metadatas, documents, embeddings, str(collection_id))
         coll = self._get_collection(collection_id)
         self._manager.hint_use_collection(collection_id, t.Operation.ADD)
-        validate_batch(
-            (ids, embeddings, metadatas, documents, uris),
-            {"max_batch_size": self.get_max_batch_size()},
+
+        self._validate_embedding_record_set(
```

Comment:
by the time `add` hits the server, `embeddings` should not be `None`. We check it by type, but not at runtime. `_validate_embedding_record_set` as currently written does not check for the presence of `embeddings` when we expect to.

Reply:
_validate_embedding_record_set invokes validate_embeddings. inside that function, we do make sure that embeddings is not None by ```if not isinstance(embeddings, list):```. guess we can be more explicit, what do you think?

Reply:
embeddings doesn't get checked if it is None